go.mod

```diff
@@ -59,7 +59,7 @@ require (
 	github.com/openshift/client-go v0.0.0-20220525160904-9e1acff93e4a
 	github.com/openshift/cloud-credential-operator v0.0.0-00010101000000-000000000000
 	github.com/openshift/console-operator v0.0.0-20220407014945-45d37e70e0c2
-	github.com/openshift/hive/apis v0.0.0-20231116161336-9dd47f8bfa1f
+	github.com/openshift/hive/apis v0.0.0-20240131003045-5fbe0d158ba9
```

Comment:
For next time, we should have a more clear step-by-step on this part of the doc:
```
Revendor Hive in the RP to match the version that we intend to upgrade.
Example PR can be found https://github.com/Azure/ARO-RP/pull/3119 
```
Is this the only field that gets updated for the revendor?

Reply:
Yes, that example shows updates to kube-api which I think may have been required when updating hive then.

[Wiki Page Updated](https://msazure.visualstudio.com/AzureRedHatOpenShift/_wiki/wikis/ARO.wiki/366470/Hive-Deploy#)